sdk/how-to-guides/manage-storage.md

```diff
@@ -0,0 +1,232 @@
+> See first: [Juju | How to Manage Storage](https://juju.is/docs/juju/manage-storage)
+
+**Contents:**
+
+- [Implement storage management](#heading--implement-storage-management)
+- [Test storage management](#heading--test-storage-management)
+
+
+<a href="#heading--implement-storage-management"><h2 id="heading--implement-storage-management">Implement storage management</h2></a>
+
+- [Declare the storage in `charmcraft.yaml`](#heading--declare-the-storage-in-charmcraft-yaml)
+- [Observe the attached event and define an event handler](#heading--observe-the-attached-event-and-define-an-event-handler)
+
+
+<a href="#heading--declare-the-storage-in-charmcraft-yaml"><h3 id="heading--declare-the-storage-in-charmcraft-yaml">Declare the storage in `charmcraft.yaml`</h3></a>
+
+To define the storage that can be provided to the charm, define a `storage` section in `charmcraft.yaml` that lists the storages and information about each storage. For example, for a transient filesystem storage mounted to `/cache/` that is at least 1GB in size:
+
+```yaml
+storage:
+  local-cache:
+      type: filesystem
+      description: Somewhere to cache files locally.
+      location: /cache/
+      minimum-size: 1G
+      properties:
+          - transient
+```
+
+> See more: [File `charmcraft.yaml` > `actions`](/t/7132#heading--storage)
+
+<a href="#heading--observe-the-attached-event-and-define-an-event-handler"><h3 id="heading--observe-the-attached-event-and-define-an-event-handler">Observe the attached event and define an event handler</h3></a>
+
+In the `src/charm.py` file, in the `__init__` function of your charm, set up an observer for the attached event associated with your storage and pair that with an event handler, typically a holistic one. For example:
```

Comment:
I'd remove "attached" (it's not clear to me what it is adding).

Reply:
To distinguish between the attached and detaching events. Maybe 'storage-attached'?